### ISSUE TITLE
file and overwrite parameters are ignored

### ISSUE THREAD

**Issue by ryandamen** [nextflow-io/nf-prov#20]
Hello, 

I tried to use this in one of my pipelines and ran into some issues during testing the example snippets in the README: 
```groovy
plugins {
  id 'nf-prov'
}

prov {
  enabled = true
  formats {
    legacy {
      file = 'manifest.json'
      overwrite = true
    }
  }
}
```

The first run was successful but while rerunning the same command the overwrite parameter appears to be ignored and returned:
```bash
$ nextflow run main.nf -profile dev -stub 
N E X T F L O W  ~  version 23.10.0
Launching `main.nf` [ecstatic_chandrasekhar] DSL2 - revision: 0af3c1836a
Provenance manifest already exists: /mnt/WORKSPACE/xxxx_workspace/projects/test_pipeline/manifest.json
```

Editing `file='manifest.json'`  to `file='test/manifest.json'` resulted in the same error. Deleting the manifest and rerunning like this also didn't put the `manifest.json` within the `test` folder. This leads me to believe that the file parameter is also ignored.

Did I make a mistake somewhere or do you see something similair?

Kind regards,
Ryan

**Comment by bentsherman** [comment]
Hello Ryan, thanks for pointing it out. I updated the README without noting that this setting would be available in 1.2.0, which I hadn't released yet. I was going to wait for some other in-progress features before releasing, but since that might take a while, I went ahead and cut a new release.

Please try again with the latest plugin version 1.2.0 and let me know if it doesn't work.

**Comment by ryandamen** [comment]
I tested it out and it works as expected now, Thanks!
